packages/fxa-settings/src/components/UnitRowSecondaryEmail/index.tsx

```diff
@@ -1,80 +1,235 @@
-        </p>
-        <p className="text-grey-400 text-xs mt-2">
-          Note: a secondary email won't restore your information—you'll need
-          a{' '}
-          <a className="link-blue" href="#recovery-key">
-            recovery key
-          </a>{' '}
-          for that.
-        </p>
+        <SecondaryEmailDefaultContent />
 
         {modalRevealed && (
           <Modal
             onDismiss={hideModal}
-            onConfirm={onSecondaryEmailConfirm}
+            onConfirm={resendPrimaryEmailCodeFromModal}
             headerId={modalHeaderId}
             descId={modalDescId}
           >
             <h2
               id={modalHeaderId}
               className="font-bold text-xl text-center mb-2"
+              data-testid={modalHeaderId}
             >
               Verify primary email first
             </h2>
-            <p className="text-center" id={modalDescId}>
-              Before you can add a secondary email, you must verify your
-              primary email. To do this, you'll need access to{' '}
-              {primaryEmail}
+            <p
+              className="text-center"
+              id={modalDescId}
+              data-testid={modalDescId}
+            >
+              Before you can add a secondary email, you must verify your primary
+              email. To do this, you'll need access to {primaryEmail}
             </p>
           </Modal>
         )}
+      </UnitRow>
+    );
+  };
+
+  const UnitRowSecondaryEmailContentAndActions = ({
+    secondaryEmailObj,
+    index,
+  }: UnitRowSecondaryEmailContentAndActionsProps) => {
+    const secondaryEmail = secondaryEmailObj.email;
+    const secondaryEmailIsVerified = secondaryEmailObj.verified;
 
-        {alertBarRevealed && (
+    const resendSecondaryEmailCodeHandler = () => {
+      resendSecondaryEmailCode({
+        variables: { input: { email: secondaryEmail } },
+      });
+      revealAlertBar();
+    };
+
+    return (
+      <>
+        {alertBarRevealed && (data || error) && (
           <AlertBar onDismiss={hideAlertBar}>
-            <p>Check the inbox for {primaryEmail} to verify your primary email.</p>
+            {data && (
+              <p>
+                Check the inbox for {primaryEmail} to verify your secondary
+                email.
+              </p>
+            )}
+            {error && <p>Error text TBD. {error.message}</p>}
           </AlertBar>
         )}
-      </UnitRow>
+        <div className="mobileLandscape:flex unit-row-multi-row">
+          <div className="unit-row-content" data-testid="unit-row-content">
+            <p className="font-bold" data-testid="unit-row-header-value">
+              {secondaryEmail}
+              {!secondaryEmailIsVerified && (
+                <span
+                  data-testid="unverified-text"
+                  className="uppercase block text-orange-600 font-bold text-xs"
+                >
+                  unverified
```

Comment:
Nit: this might be visually uppercase but should prob be capitalized

Reply:
Meh, why?

Reply:
Proper grammar? 😛